### ISSUE TITLE
Add ability to list assetpairs 

### ISSUE THREAD

**Issue by pangoSE** [polyrabbit/token-ticker#10]
e.g. by parsing this from kraken: https://api.kraken.com/0/public/AssetPairs

**Comment by polyrabbit** [comment]
Yes, I'm also planning to add this feature, but got a bit busy recently.

It seems urgent now as your issue on #11.
